README.md

```diff
@@ -14,7 +14,7 @@ application developers and computer science researchers.
 
 The Open MPI documentation can be viewed in the following ways:
 
-1. Online at https://docs.open-mpi.org/
+1. Online at https://docs.open-mpi.org/en/v5.0.0/
```

Comment:
I would not mention the `en/v5.0.0/` link here.  If you surf to https://docs.open-mpi.org/, you'll get redirected to the latest docs.  If you want to see a different version, you can use the navigation picker to pick the specific version you want.

If you list `v5.0.0` here, you'll have to update this link every time you do a new release.

Reply:
done, I dropped the commit